apps/api/src/common/guards/api-key/api-key.guard.ts

```diff
@@ -9,7 +10,19 @@ export class ApiKeyGuard implements CanActivate {
     const apiKey = configService.getOrThrow<string>('SERVER_API_KEY');
     const request = context.switchToHttp().getRequest();
 
-    const authHeader = request.headers['authorization'];
+    // Get auth header incase of http request
```

Comment:
Did you see if this works for the REST API too(POST request in controller)?

Reply:
Yes, @vikas-kyatannawar ,
I verified both the cases for rest request as well, It is working fine for it.

![image](https://github.com/OsmosysSoftware/osmo-notify/assets/114974207/3e3ce710-86cb-41a1-9042-dd1ec958e7ce)